ChangeLog

```diff
@@ -60,6 +60,9 @@ modules are added.
 
   Closes #3619
 
+* New checker * ``unnecessary-dict-indexing``. Emitted when iterating over dictionary items (key-value pairs) and
+  then indexing the same dictionary with the key within loop body.
```

Comment:
```suggestion
* New checker ``unnecessary-dict-indexing``. Emitted when iterating over dictionary items (key-value pairs) and
  accessing the value by index lookup.
```
Maybe something like this? This would be more consistent with the help text.